stories/Organism/Images/Images/Images.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useEffect } from "react";
+import "./images.scss";
+import viewport from '../../../assets/js/viewport';
+import { Imagecaption } from "../../../Molecules/Imagecaption/Imagecaption";
+
+export const size_options = {
+  Wide: "medium-12",
+  Medium: "medium-7",
+  Portrait: "medium-4",
+};
+
+export const size_options2 = {
+  Wide: "medium-7",
+  Medium: "medium-7",
+  Portrait: "medium-3",
+};
+
+export const size_options3 = {
+  Wide: "",
+  Medium: "",
+  Portrait: "image-section--portrait",
+};
+
+export const Images = ({ imagelg, imagemd, imagesm, alt, name, label, paragraph, ...args }) => {
+  useEffect(() => {
+    viewport('.image__cart');
+    viewport('.image__description');
+  }, [])
+  return (
+    <div className={["grid-x", "image-section", `${size_options3[`${args.size}`]}`].join(" ")}>
+      <div className={["cell", "image__cart", "scale-up", `${size_options[`${args.size}`]}`].join(" ")}>
```

Comment:
@Shubham-Sharma-77 Still classname is not correct as it should be image-section__cart